tests/experimental/datasources/test_pandas_filesystem_datasource.py

```diff
@@ -418,24 +419,17 @@ def test_get_batch_list_from_fully_specified_batch_request(
         name="csv_asset",
         batching_regex=r"yellow_tripdata_sample_(?P<year>\d{4})-(?P<month>\d{2})\.csv",
     )
-    path = "yellow_tripdata_sample_2018-04.csv"
-    request = asset.build_batch_request({"year": "2018", "month": "04", "path": path})  # type: ignore[attr-defined]
+    request = asset.build_batch_request({"year": "2018", "month": "04"})  # type: ignore[attr-defined]
     batches = asset.get_batch_list_from_batch_request(request)  # type: ignore[attr-defined]
     assert len(batches) == 1
     batch = batches[0]
     assert batch.batch_request.datasource_name == pandas_filesystem_datasource.name
     assert batch.batch_request.data_asset_name == asset.name  # type: ignore[attr-defined]
-    assert batch.batch_request.options == {
-        "year": "2018",
-        "month": "04",
-        "path": pathlib.Path(path),
-    }
-    assert batch.metadata == {
-        "year": "2018",
-        "month": "04",
-        "base_directory": pandas_filesystem_datasource.base_directory,
-        "path": pathlib.Path(path),
-    }
+
+    path = "yellow_tripdata_sample_2018-04.csv"
+    assert batch.batch_request.options == {"path": path, "year": "2018", "month": "04"}
+    assert batch.metadata == {"path": path, "year": "2018", "month": "04"}
```

Comment:
`non-blocking`
This feels like we are going a bit backwards. Why is this no a `str` rather then a `pathlib.Path` object?

Reply:
@Kilo59 My understand is that this is metadata, where all values were strings up to now, because they resulted from partitioning the data reference using RegEx.  Whether or the `path` addition should also be a string may need some discussion.  I would say that it should be a string, because to the user the data references for files are strings.